addon/PreviewProvider.js

```diff
@@ -370,7 +378,7 @@ PreviewProvider.prototype = {
    */
   insertMetadata(links, metadataSource) {
     const linksToInsert = links.map(link => Object.assign({}, link, {
-      expired_at: (this.options.metadataTTL) + Date.now(),
+      expired_at: (this._getMetadataTTL()) + Date.now(),
```

Comment:
nit: calling this function for each insert seems to be unnecessary. I think it's fine as it's currently just running as an experiment, we may want to call it once and store the TTL at somewhere once the experiment is done.